akka-http-core/src/main/java/akka/http/javadsl/model/HttpHeader.java

```diff
@@ -26,7 +26,7 @@
     public abstract String lowercaseName();
 
     /**
-     * Returns true iff nameInLowerCase.equals(lowercaseName()).
+     * Returns true if nameInLowerCase.equals(lowercaseName()).
```

Comment:
Interesting, didn't know that 😃 
What about `Returns only true if...` ? 
I mean `Boolean` kind of implies that it returns false otherwise.